src/components/card/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import * as S from './styled';
+
+interface CardProps {
+  uid: number;
+  contents: String;
+  onClick: () => void;
+}
+
+export default function Card({ contents, onClick }: CardProps) {
+  const choice = () => {
+    onClick();
+  };
+
+  return (
+    <S.CardWrapper onClick={choice}>

```

Comment:
P3;

아래처럼 바로 받아서 사용하면 에러가 발생하나요?
```suggestion
  return (
    <S.CardWrapper onClick={onClick}>
```

혹은
```suggestion

  return (
    <S.CardWrapper onClick={() => { onClick() }}>

```

Reply:
에러가 발생하지 않습니다. 
제시해주신 방법대로 코드를 줄이는 게 훨씬 간단해보이네요.